app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationTopAppBar.kt

```diff
@@ -97,6 +99,22 @@ fun ConversationScreenTopAppBar(
                     overflow = TextOverflow.Ellipsis,
                     modifier = Modifier.weight(weight = 1f, fill = false)
                 )
+                if (conversationInfoViewState.mlsVerificationStatus == ConversationVerificationStatus.VERIFIED) {
+                    Icon(
+                        modifier = Modifier.padding(start = dimensions().spacing4x),
+                        painter = painterResource(id = R.drawable.ic_certificate_valid_mls),
+                        tint = colorsScheme().mlsVerifiedIconColor,
+                        contentDescription = stringResource(R.string.content_description_mls_certificate_valid)
+                    )
+                }
+                if (conversationInfoViewState.proteusVerificationStatus == ConversationVerificationStatus.VERIFIED) {
+                    Icon(
+                        modifier = Modifier.padding(start = dimensions().spacing4x),
+                        painter = painterResource(id = R.drawable.ic_certificate_valid_proteus),
+                        tint = colorsScheme().mlsVerifiedIconColor,
+                        contentDescription = stringResource(R.string.content_description_mls_certificate_valid)
```

Comment:
I think the text here should refer to protues text.

Reply:
done